operations/addresses.md

```diff
@@ -0,0 +1,238 @@
+        }
+    ],
+    "Cursor": "fc7b2df3-de66-48a6-907d-af4600ecd892"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Addresses` | array of [Account address](#account-address) | required | The collection of Account addresses, containing address and account information. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older Account address. |
+
+#### Account address
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the address. |
+| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
+| `AccountType` | string | required | A discriminator specifying the [type of account](#account-type), e.g. customer or company. |
+| `Line1` | string | optional | First line of the address. |
+| `Line2` | string | optional | Second line of the address. |
+| `City` | string | optional | The city. |
+| `PostalCode` | string | optional | Postal code. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude. |
+| `Longitude` | number | optional | The longitude. |
+
+#### Account type
+
+* [Company](companies.md#company)
+* [Customer](customers.md#customer)
+
+## Add addresses
+
+Adds one or more new addresses to the system and assign them to specified accounts.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/addresses/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Addresses": [
+        {
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Addresses` | array of [Account address adds](#account-address-add-parameters), max 1000 items | required | Collection of addresses to be created. |
+
+#### Account address add parameters
```

Comment:
The `Account address adds` doesn't roll of my tongue, in other operations we usually have it like this.
```suggestion
| `Addresses` | array of [Account address parameters](#account-address-parameters), max 1000 items | required | Collection of addresses to be created. |

#### Account address parameters
```